server/__tests__/integration/amplify.test.js

```diff
@@ -19,7 +19,7 @@ describe('/api/amplify/:zipcode', () => {
     })
 })
 
-describe('/api/amplify/', () => {
+describe.skip('/api/amplify/', () => {
```

Comment:
We commented out the associated endpoint in `server/routes/api/amplify.js` so these tests were failing with `404` response statuses.